conf/defaults.ini

```diff
@@ -248,7 +248,7 @@ enabled = false
 allow_sign_up = true
 client_id = some_id
 client_secret = some_secret
-scopes = user:email
+scopes = user:email,read:org
```

Comment:
you need to change sample.ini as well.

Reply:
scope values are usually separated with a space character, not a comma

Reply:
@DanCech github normalizes them as comma separated, if i'm not mistaken.

Reply:
@torkelo [sample.ini](https://github.com/grafana/grafana/blob/master/conf/sample.ini#L235) seems fine

Reply:
https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/ read the note

Reply:
@DanCech sure, I can use space delimited scopes if needed.  
`SplitString` function handles both cases anyway 👍